packages/graphql-language-service-server/src/startServer.js

```diff
@@ -17,67 +17,88 @@ import {
 } from './MessageProcessor';
 
 import {
+  IPCMessageReader,
   IPCMessageWriter,
   SocketMessageReader,
   SocketMessageWriter,
 } from 'vscode-jsonrpc';
 
-type ServerOptions = {
+type Options = {
   port?: number,
+  method?: string,
+  configDir?: string,
```

Comment:
`index.js` sounds like a great place for it. Keeps the local type consistent with the intentions of the code.